### ISSUE TITLE
Add commit method to WizardSettings

### ISSUE THREAD

**Issue by PhoneixS** [cjwizard/cjwizard#25]
Now, we have a rollBack() methos in WizardSettings interface.

I think that we can also have a commit() method. For example if a page have an action that must be done only once, when you have done it you add a new settings value and commit the changes so although he goes back immediately after the action have been done and then forward again, you can know that the action is already done.

Is this a good idea?
